lua/hop/jump_target.lua

```diff
@@ -80,6 +80,10 @@ local function mark_jump_targets_line(buf_handle, win_handle, regex, line_contex
     elseif direction_mode.direction == hint.HintDirection.BEFORE_CURSOR then
       -- we want to change the end
       shifted_line = shifted_line:sub(1, col - col_offset)
+    elseif direction_mode.direction == hint.HintDirection.AFTER_CURSOR_LINE then
+      return
```

Comment:
this can be considered to be an error caused by developers that with HintDirection.AFTER_CURSOR_LINE cursor_pos should never be given. thus i really want to to throw an error here.